src/components/NcIconSvgWrapper/NcIconSvgWrapper.vue

```diff
@@ -285,6 +285,13 @@ export default {
 		vertical-align: text-bottom;
 	}
 
+	// Icon svg wrapper
+	span {
+		display: flex;
+		justify-content: center;
+		align-items: center;
```

Comment:
No other flex needed, the span normally should just wrap the svg. But as we enforce a line height its at least that height. So just removing it will solve the problem.

```suggestion
		line-height: 0;
```

Reply:
Who are you with all your wizardry! :fearful: